modules/docs-tests/src/test/scala/sclicheck/DocTests.scala

```diff
@@ -1,17 +1,16 @@
 package sclicheck
 
-class DocTests extends munit.FunSuite {
+import os.Path
```

Comment:
NIT Let's not import that one? It doesn't save many characters, and makes it clear that those paths originate from os-lib, rather than NIO.

Reply:
yep, yep, wasn't intentional, my bad
should be fixed.